### PR TITLE
Move Helm from Beta to Stable

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

**Comment by sdake** [comment]
@costinm - I'd like your technical opinion if you think we have reached stable for Helm before proceeding with further approval of this PR.

Cheers
-steve

**Comment by sdake** [comment]
Removed APPROVED label since I submitted this PR and it autoapplies if the author is in the OWNERS file.

Cheers
-steve

**Comment by costinm** [comment]
Well - since we still have the ordering issue, and the options in values.yaml are still a bit all over the place - I would keep beta.

At the same time - it is pretty stable and works well. So could mark it as stable too. 

Maybe wait another release, until we sort out pre-install and clean up more the settings ? We still have plenty of undocumented.

**Comment by sdake** [comment]
@costinm right - 1.1 makes sense to me - I think we can tackle any remaining construction by that release.

Will leave PR open.

Cheers
-steve

File: content/about/feature-stages/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Below is our list of existing features and their current phases. This informatio
 | [Custom Mixer Build Model](https://github.com/istio/istio/wiki/Mixer-Compiled-In-Adapter-Dev-Guide) | Alpha
 | [Out of Process Mixer Adapters](https://github.com/istio/istio/wiki/Out-Of-Process-gRPC-Adapter-Dev-Guide) | Alpha
 | Enable API attributes using an IDL | Alpha
-| [Helm](/docs/setup/kubernetes/helm-install/) | Beta
+| [Helm](/docs/setup/kubernetes/helm-install/) | Stable
```
[the remark this answers is not in the document]
Comment (sdake) [reply content/about/feature-stages/index.md]: @andraxylia - @costinm has implemented a conditional in the Helm chart so once Helm 2.10 is available, 1.0 will be usable with 2.10 without extra steps using a CRD hook.

**Comment by geeknoid** [comment]
Is this PR still needed?

**Comment by geeknoid** [comment]
Closing due to inactivity.

**Pull request closed without merging**